GPU/Vulkan/DrawEngineVulkan.cpp

```diff
@@ -359,12 +352,9 @@ void DrawEngineVulkan::BeginFrame() {
 
 void DrawEngineVulkan::EndFrame() {
 	FrameData *frame = &GetCurFrame();
-	stats_.pushUBOSpaceUsed = (int)frame->pushUBO->GetOffset();
-	stats_.pushVertexSpaceUsed = (int)frame->pushVertex->GetOffset();
-	stats_.pushIndexSpaceUsed = (int)frame->pushIndex->GetOffset();
-	frame->pushUBO->End();
-	frame->pushVertex->End();
-	frame->pushIndex->End();
+	stats_.pushUBOSpaceUsed = 0; // (int)pushUBO->GetOffset();
+	stats_.pushVertexSpaceUsed = 0; // (int)frame->pushVertex->GetOffset();
+	stats_.pushIndexSpaceUsed = 0; // (int)frame->pushIndex->GetOffset();
```

Comment:
I think these stats are still TODO?  Or should we remove?

-[Unknown]

Reply:
First one is wildly inaccurate since we have a lot of other stuff in that buffer. I'll fix up the other ones.